src/lookup-table/controllers/ImportCsv.controller.ts

```diff
@@ -3,29 +3,31 @@ import { parseStream } from 'fast-csv'
 import { LookupTableService } from '../services'
 
 const ImportCsvController = async (request: FastifyRequest, reply: FastifyReply) => {
-  const data = await request.file()
-  const tableNameLabel: string = (data.fields.tableName as any).value
-  const lookupTableService = await LookupTableService({ tableNameLabel })
+  const data = await request.files()
+  const { tableName } = request.query as any
```

Comment:
Because of the "files" change, I'm now sending the "tableName" as url query rather than part of formData. Front-end updated accordingly.